unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -867,15 +867,15 @@ PlayerSettings:
   m_RenderingPath: 1
   m_MobileRenderingPath: 1
   metroPackageName: sample
-  metroPackageVersion: 
+  metroPackageVersion: 1.0.0.0
   metroCertificatePath: 
   metroCertificatePassword: 
   metroCertificateSubject: 
   metroCertificateIssuer: 
   metroCertificateNotAfter: 0000000000000000
   metroApplicationDescription: sample
   wsaImages: {}
-  metroTileShortName: 
+  metroTileShortName: Croquet for Unity
```

Comment:
Checking: is the different name appropriate here?

Reply:
ohhh I will take this out. 
Metro Tile is a Windows 8 interface thing